cypress/integration/project.spec.js

```diff
@@ -32,243 +31,513 @@ const checkTodoList = (todos = []) => {
+      checkTodoList([todo1, todo2]);
+      cy.get(TODO_ADD_INPUT_SELECTOR).should('have.value', '');
+    });
   });
 
-  it('Os itens da lista de tarefas devem ser ordenados por ordem de criação', () => {
-    const todo1 = {
-      content: 'minha tarefa'
-    };
+  describe('Os itens da lista de tarefas devem ser ordenados por ordem de criação', () => {
+    it('Três itens serão criados na lista e será checado se eles estão ordenados por ordem de criação - ou seja, primeiro o primeiro item criado, depois o segundo, e assim por diante', () => {
+      const todo1 = {
+        content: 'minha tarefa',
+      };
 
-    const todo2 = {
-      content: 'minha outra tarefa'
-    };
+      const todo2 = {
+        content: 'minha outra tarefa',
+      };
 
-    const todo3 = {
-      content: 'minha terceira tarefa'
-    };
+      const todo3 = {
+        content: 'minha terceira tarefa',
+      };
 
-    addTodo(todo1);
-    checkTodoList([todo1]);
+      addTodo(todo1);
+      checkTodoList([todo1]);
 
-    addTodo(todo2);
-    checkTodoList([todo1, todo2]);
+      addTodo(todo2);
+      checkTodoList([todo1, todo2]);
 
-    addTodo(todo3);
-    checkTodoList([todo1, todo2, todo3]);
+      addTodo(todo3);
+      checkTodoList([todo1, todo2, todo3]);
+    });
   });
 
-  it('Ao passar o mouse sobre qualquer botão o ícone do mouse deve mudar para uma mãozinha', () => {
-    const todos = [{
-      content: 'tarefinha facin de fazer'
-    }, {
-      content: 'tarefa mais hard'
-    }];
+  describe('Ao clicar em um item da lista, altere a cor de fundo do item para cinza rgb(128,128,128)', () => {
+    it('Será verificado que, ao se carregar a página, os itens da lista **não tem** o estilo CSS `background-color: rgb(128, 128, 128)`', () => {
+      const todos = [
+        {
+          content: 'uma tarefa qualquer',
+        },
+        {
+          content: 'uma outra tarefa atoa',
+        },
+      ];
+
+      addTodos(todos);
+      checkTodoList(todos);
+
+      cy.get(TODO_LIST_LINE_SELECTOR)
+        .first()
+        .should('not.have.css', 'background-color', 'rgb(128, 128, 128)');
+    });
 
-    addTodos(todos);
-    checkTodoList(todos);
+    it('Será verificado que, ao se clicar em um item da lista, ele passa a ter o estilo CSS `background-color: rgb(128, 128, 128)`', () => {
+      const todos = [
+        {
+          content: 'uma tarefa qualquer',
+        },
+        {
+          content: 'uma outra tarefa atoa',
+        },
+      ];
+
+      addTodos(todos);
+      checkTodoList(todos);
+
+      cy.get(TODO_LIST_LINE_SELECTOR)
+        .first()
+        .click()
+        .then(($li) => {
+          expect($li).to.have.css('background-color', 'rgb(128, 128, 128)');
+        });
+    });
+  });
 
-    cy.get(TODO_LIST_LINE_SELECTOR).first().trigger('hover').then(($li) => {
-      expect($li).to.have.css('cursor', 'pointer');
+  describe('Não deve ser possível selecionar mais de um elemento da lista ao mesmo tempo.', () => {
```

Comment:
Os describes dos requisitos tem que ser exatamente iguais ao mapeamento do `.trybe/requirements.json`. Então, mesmo um ponto a mais faz diferença. Cuidado :grimacing: 

```suggestion
  describe('Não deve ser possível selecionar mais de um elemento da lista ao mesmo tempo', () => {
```

Reply:
Passou batido, valeu 😬